src/js/disability-benefits/686/config/form.js

```diff
@@ -168,9 +176,139 @@ const formConfig = {
                   address: address.schema(fullSchema686),
                   claimantEmail
                 }
+              }
+            }
+          }
+        }
+      }
+    },
+    householdInformation: {
+      title: 'Household Information',
+      pages: {
+        marriageInfo: {
+          title: 'Marriage history',
+          path: 'household/marriage-info',
+          uiSchema: {
+            maritalStatus: {
+              'ui:title': 'What’s your marital status?',
+              'ui:widget': 'radio'
+            },
+            marriages: {
+              'ui:title': 'How many times have you been married?',
+              'ui:widget': ArrayCountWidget,
+              'ui:field': 'StringField',
+              'ui:required': (form) => !!_.get('maritalStatus', form)
+              && form.maritalStatus !== 'Never Married',
+              'ui:options': {
+                showFieldLabel: 'label',
+                keepInPageOnReview: true,
+                expandUnder: 'maritalStatus',
+                expandUnderCondition: (status) => !!status
+                && status !== 'Never Married'
               },
+              'ui:errorMessages': {
+                required: 'You must enter at least 1 marriage'
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            required: ['maritalStatus'],
+            properties: {
+              maritalStatus,
+              marriages
+            }
+          }
+        },
+        marriageHistory: {
+          title: (form, { pagePerItemIndex }) => getMarriageTitleWithCurrent(form, pagePerItemIndex),
+          path: 'household/marriages/:index',
+          showPagePerItem: true,
+          arrayPath: 'marriages',
+          uiSchema: {
+            marriages: {
+              items: {
+                'ui:options': {
+                  updateSchema: (form, schema, uiSchema, index) => {
+                    return {
+                      title: getMarriageTitleWithCurrent(form, index)
+                    };
+                  }
+                },
+                spouseFullName: {
+                  'ui:options': {
+                    updateSchema: (form, schema, uiSchema, index) => {
+                      const spouse = isCurrentMarriage(form, index) ? 'Spouse’s' : 'Former spouse’s';
+                      return _.merge(schema, {
```

Comment:
One thing to note with this is that you're going to return a new object each time, so the schema is going to get updated on every change and cause this field to be re-rendered. It may not be noticeable, but in the past we've either used reselect or generated the different outputs outside of this function switched between them here.